package.json

```diff
@@ -10,6 +10,7 @@
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.2.0",
+    "rxjs": "^6.5.3",
```

Comment:
is required for you at the moment to have `rxjs` installed?

Reply:
at the moment, no. I will uninstall it for now